Zend/zend_vm_execute.skl

```diff
@@ -5,10 +5,19 @@
 # pragma GCC optimize("no-gcse")
 # pragma GCC optimize("no-ivopts")
 #endif
+#ifdef _WIN64
+/* See save_xmm_x86_64_ms_masm.asm */
+void {%EXECUTOR_NAME%}_ex_real(zend_execute_data *ex)
+#else
 ZEND_API void {%EXECUTOR_NAME%}_ex(zend_execute_data *ex)
+#endif
 {
 	DCL_OPLINE
 
+#if defined(__GNUC__) && defined(__aarch64__)
+	__asm__ __volatile__ (""::: "v8","v9","v10","v11","v12","v13","v14","v15");
+#endif
+
```

Comment:
Do we need this for HYBRID VM?

Reply:
I think we do. The reason we don't save registers in HYBRID VM JIT'ed code is that `execute_ex` does it in `HYBRID_JIT_GUARD()` before calling handlers, but this doesn't save these registers.

Reply:
Yes. If I remember correctly, the test also originally failed on circleci which is Linux+arm64+gcc so should use hybrid VM.